packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/request.js

```diff
@@ -118,5 +118,25 @@ describe('plugin-meetings', () => {
         assert.equal(requestParams.body.hand.raised, true);
       });
     });
+
+    describe('#lowerAllHands', () => {
+      it('sends a PATCH to the locus endpoint', async () => {
+        const locusUrl = url1;
+        const memberId = 'test1';
+
+        const options = {
+          requestingParticipantId: memberId,
+          locusUrl,
+        };
+
+        await membersRequest.lowerAllHandsMember(options);
+        const requestParams = membersRequest.request.getCall(0).args[0];
+
+        assert.equal(requestParams.method, 'PATCH');
+        assert.equal(requestParams.uri, `${locusUrl}/controls`);
+        assert.equal(requestParams.body.hand.raised, false);
+        assert.equal(requestParams.body.requestingParticipantId, memberId);
```

Comment:
suggestion: Just do a `assert.deepEqual` for the entire `requestParams` object - there shouldn't be unexpected items in this (it hasn't yet been through the request interceptors to have modifications added) so we should be strict about what is being used for the request.